app/livechat/client/views/app/livechatCustomFields.js

```diff
@@ -49,6 +56,7 @@ Template.livechatCustomFields.events({
 	},
 });
 
-Template.livechatCustomFields.onCreated(function() {
-	this.subscribe('livechat:customFields');
+Template.livechatCustomFields.onCreated(async function() {
+	this.customFields = new ReactiveVar([]);
+	await loadCustomFields(this);
```

Comment:
no need to `await` since there is no code after the call

```suggestion
	loadCustomFields(this);
```